src/js/utils/captions.js

```diff
@@ -250,10 +250,11 @@ export const areSubsOn = (el) => {
  *
  * This was originally in media-captions-button.
  *
- * @param {HTMLElement} el - An HTMLElement that has caption related attributes on it.
+ * @param {HTMLElement & { mediaSubtitlesList?: any[] }} el - An HTMLElement that has caption related attributes on it.
  * @param {boolean} [force] - An optional boolean that will force captions to the given state. True for on and false for off
+ * @param {boolean} [noSubtitlesPref = false] - An optional boolean that will force captions to the given state. True for on and false for off
  */
-export const toggleSubsCaps = (el, force) => {
+export const toggleSubsCaps = (el, force, noSubtitlesPref = false) => {
```

Comment:
I went with the negative phrasing so the "standard usage" for this method wouldn't require a refactor. I could remove the default if that feels bad, since `undefined` is falsy?

Reply:
No longer an issue with the refactor that adds the toggle request event.